admin/common_auth/views.py

```diff
@@ -47,3 +55,45 @@ def get_success_url(self):
 def logout_user(request):
     logout(request)
     return redirect('auth:login')
+
+
+class RegisterUser(SuperUser, FormView):
+    form_class = UserRegistrationForm
+    template_name = 'register.html'
+
+    def form_valid(self, form):
+        osf_id = form.cleaned_data.get('osf_id')
+        try:
+            osf_user = User.load(osf_id)
```

Comment:
Unless you expect `osf_user = User.load(osf_id)` to throw an AttributeError I'd pull it out of the try. It just limits the possibility you'll catch an exception that you weren't expecting.